src/main/java/com/appium/capabilities/Capabilities.java

```diff
@@ -13,11 +13,11 @@
 import java.util.Map;
 import java.util.Set;
 
-public class CapabilityManager {
-    private static CapabilityManager instance;
+public class Capabilities {
```

Comment:
I would rather keep this as-is. Capabilities in the Appium world denotes a different thing altogether.